typescript/packages/subsurface-viewer/src/components/Map.tsx

```diff
@@ -1535,7 +1552,7 @@ function buildDeckGlViews(views: ViewsType | undefined, size: Size): View[] {
         return [
             new OrthographicView({
                 id: "main",
-                controller: { doubleClickZoom: false },
+                controller: null,
```

Comment:
Does the "double-click to move camera to point" functionality work after removing this?

Reply:
This is a dummy view when in case no valid view is provided.
controller = null means no interaction at all.

Whenever anything is displayed (ie. this view is not used), double click still works correctly ;)